packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -114,6 +126,14 @@ export default class StandardModal extends React.Component<Props> {
             </ModalDialog>
         );
     }
+
+    render() {
+        return (
+            <UniqueIDProvider>
```

Comment:
We may actually want to set `mockOnFirstRender={true}` for the use case of rendering modals on page load.

I just realized that for the other modals we have `aria-labelledby="wb-modal-title"` but we haven't set the id on the title inside the modal.  This case is more interesting b/c the `<Title>` element is user specified.  I wonder if we could use the Context API in React 16 and have the modal provide a unique id for the title and then the `Title` component could use a context consumer to grab that id.

cc @somewhatabstract

Reply:
I would rather we did something less magical. Why not allow the user to specify the id prop for the title on the modal props whenever why provide a `Title`?

Reply:
@kevinbarabash Maybe you could comment on this? I think you two have more context about the use-cases here. We could also provide an optional prop so a user could specify the id, and assign a unique one if not, but maybe that's too fancy.

Reply:
A `titleId` prop is probably a good start place.  I worry about how we might lint for this.  If we can't lint for it then maybe we'll have to resort to magic.

Reply:
We have a similar need in the `Tooltip` component, so perhaps we can follow the same pattern from there to be consistent?